### PR TITLE
Addressing "Neighbor table overflow" issue.

### PR DESCRIPTION
**Pull request by moes1**
Changing gc_thresh settings to address "Neighbor table overflow" issue. For an example discussing this see this blog post: https://openai.com/blog/scaling-kubernetes-to-2500-nodes/

Values were chosen to match defaults recommended by AKS: https://docs.microsoft.com/en-us/azure/aks/custom-node-configuration#socket-and-network-tuning

### CODE REVIEW

**Comment by moes1** [comment]
Removed the sunrpc tcp_max_slot_table_entries fix because it doesn't survive reboots unless we add a service to modprobe sunrpc. Should we add that?

**Comment by moes1** [comment]
I have decided to modify the PR to add in a systemd service to start sunrpc at startup just to show what that would look like.

File: ubuntu/common/hpc-tuning.sh [hunk 1]
```diff
@@ -30,7 +30,37 @@ cat << EOF >> /etc/security/limits.conf
...
+[Install]
+WantedBy=multi-user.target
+EOF
+
+systemctl enable sunrpc_tcp_settings
+systemctl start sunrpc_tcp_settings
+
 
 # Install WALinuxAgent
 apt-get install python3-setuptools
```
Comment (rafsalas19) [inline ubuntu/common/hpc-tuning.sh:66]: Why is this done here rather than utils file?

**Review by rafsalas19 — APPROVED** [review]

File: ubuntu/common/hpc-tuning.sh (hunk 1 above)
Comment (moes1) [reply ubuntu/common/hpc-tuning.sh:66]: This refers to the install of python3-setuptools? That's a good point we probably should move that.

File: ubuntu/common/hpc-tuning.sh [hunk 2]
```diff
@@ -30,7 +30,37 @@ cat << EOF >> /etc/security/limits.conf
...
+echo "net.ipv4.neigh.default.gc_thresh2 = 8192" >> /etc/sysctl.conf
+echo "net.ipv4.neigh.default.gc_thresh3 = 16384" >> /etc/sysctl.conf
+echo "sunrpc.tcp_max_slot_table_entries = 128" >> /etc/sysctl.conf
+
+
+## Systemd service for starting sunrpc and adding setting parameters
+cat <<EOF >/usr/sbin/sunrpc_tcp_settings.sh
+#!/bin/bash
+
+modprobe sunrpc
```
Comment (jeseszhang1010) [inline ubuntu/common/hpc-tuning.sh:43]: Does module sunrpc have any dependencies, which need to be loaded first?

Comment (moes1) [reply ubuntu/common/hpc-tuning.sh:43]: I don't believe so. "modprobe --show-depends sunrpc" just shows: 

sunrpc.ko

File: ubuntu/common/hpc-tuning.sh (hunk 1 above)
Comment (jeseszhang1010) [reply ubuntu/common/hpc-tuning.sh:66]: Prefer separate PR, as it's not related to the major change in this PR.

**Review by jeseszhang1010 — APPROVED** [review]

**Pull request merged**